WcaOnRails/db/structure.sql

```diff
@@ -669,12 +669,15 @@ DROP TABLE IF EXISTS `bookmarked_competitions`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `bookmarked_competitions` (
   `id` bigint(20) NOT NULL AUTO_INCREMENT,
-  `competition_id` varchar(191) COLLATE utf8mb4_unicode_ci NOT NULL,
+  `competition_id` varchar(191) NOT NULL,
   `user_id` int(11) NOT NULL,
+  `created_at` datetime NOT NULL,
```

Comment:
@timreyn, FYI, I think this file may have gotten out of sync when you added this?

Reply:
Whoops, sorry!  Not sure where all of those diffs came from, but yes this looks like this file got out of sync while I was iterating on the migration.